docs/delegates/_example.md

```diff
@@ -7,6 +7,9 @@ twitter: username
 github: username
 forum: username
 
+# if you share part of the tokens to your supporters 
```

Comment:
```suggestion
# if you share part of the tokens to your supporters 
# You should explain below how do you share the tokens
```

Reply:
I added a link to the tool that several delegates use